services/web/server/src/simcore_service_webserver/projects/projects_api.py

```diff
@@ -92,11 +89,8 @@ def _is_node_dynamic(node_key: str) -> bool:
     return "/dynamic/" in node_key
 
 
-async def validate_project(app: web.Application, project: dict):
-    project_schema = app[APP_JSONSCHEMA_SPECS_KEY]["projects"]
-    await asyncio.get_event_loop().run_in_executor(
-        None, validate_instance, project, project_schema
-    )
+async def validate_project(project: ProjectDict):
+    await asyncio.get_event_loop().run_in_executor(None, parse_obj_as, Project, project)
```

Comment:
Why is this async. I don't get it? At this point you can drop all the async part. and make the .
Please also adjust usage

```suggestion
def validate_project(project: ProjectDict) -> None:
    parse_obj_as(Project, project)
```

Reply:
I tried to keep the same function, but I get it is not necessary

Reply:
same here